mne/time_frequency/tfr.py

```diff
@@ -99,6 +99,59 @@ def morlet(sfreq, freqs, n_cycles=7.0, sigma=None, zero_mean=False):
         Ws.append(W)
     return Ws
 
+def wavelet(sfreq, freqs, n_cycles=7.0, window=np.hanning):
+    """Compute wavelets for the given frequency range with the specified windowing function.
+
+        Parameters
```

Comment:
This is all indented too far by one level (Sphinx will probably complain)

Reply:
> This is all indented too far by one level (Sphinx will probably complain)
I didn't realize what you mean. do you mean that the comments is too lengthy? I have just copied it from the morlet function.

Reply:
```suggestion
    Parameters
```
same for all lines that follow

Reply:
Thanks